data.sql

```diff
@@ -4,4 +4,7 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', '2018/11/15', 2, true, 8);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', '2021/01/07', 1, false, 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', '2017/05/12', 5, true, 11);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, wight_kg) VALUES ('Charmander', '2020/02/08', 0, false, -11), 
+('Plantmon','2021/03/15', 2, true, -5.7), ('Squirtle', '1993/04/02', 3, false, -12.13), ('Angemon', '2005/06/12', 1, true, -45), 
```

Comment:
- [ ] Kindly make sure that the date of birth for Plantmon is correct. It should be 2021/11/15.